src/ContractInteract/Anchor.js

```diff
@@ -42,6 +67,168 @@ class Anchor {
     this.anchorStateRoot = this.anchorStateRoot.bind(this);
   }
 
+  /**
+   * Setup for Anchor contract. Retrieves information about the remote chain,
+   * deploys the Anchor contract, and sets its coAnchor address (if provided).
+   *
+   *
+   * @param {Web3} originWeb3 Web3 object.
+   * @param {Web3} auxiliaryWeb3 Web3 object for remote chain.
+   * @param {AnchorSetupConfig} config Anchor setup configuration.
+   * @param {Object} txOptions Transaction options.
+   *
+   * @returns {Promise<Anchor>} Promise containing the Anchor instance that
+   *                         has been set up.
+   */
+  static setup(originWeb3, auxiliaryWeb3, config, txOptions = {}) {
```

Comment:
origin and auxiliary are actually the wrong terms for the anchor. I must also deploy it on auxiliary.
So depending on where I deploy it, there is a chain where I deploy it, and a remote chain that I track. For both chains you need the web3 object if you want to know the current block number of the remote chain. But auxiliary is not necessarily the remote. Origin could as well be the remote chain.

Reply:
I changed it because one of you previous comments about `coWeb3` 😅 . So revert to previous naming of `web3` and `coWeb3` (or is there are other naming we are using for that now)?